src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Get started with Auto-telemetry with Pixie
```

Comment:
Maybe the title should focus more on what you can do with Auto-telemetry with Pixie?